incubator-tools/cmek_docai_processor/cmek_docai_processor.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    import google.cloud.documentai_v1beta3 as documentai
</pre><p>All of these imports should be at the top of the notebook and only done once.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Updated